tests/parsed_solver_01.cc

```diff
@@ -21,11 +21,18 @@
 
```

Comment:
di default il linear operator usa default_reinit della classe (che ha solo un assert) e il test in modalità debug a volte mi salta.